core/events/events_block_field_intermediate_change.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Class for an event representing an intermediate change to a block's field's
+ * value.
+ *
+ * @class
+ */
+import * as goog from '../../closure/goog/goog.js';
+goog.declareModuleId('Blockly.Events.BlockFieldIntermediateChange');
+
+import type {Block} from '../block.js';
+import type {BlockSvg} from '../block_svg.js';
+import * as registry from '../registry.js';
+import * as utilsXml from '../utils/xml.js';
+import {Workspace} from '../workspace.js';
+import * as Xml from '../xml.js';
+
+import {AbstractEventJson} from './events_abstract.js';
+import {UiBase} from './events_ui_base.js';
+import * as eventUtils from './utils.js';
```

Comment:
I thought I meant these lint warnings error but apparently I didn't -.- could you fix these before merging?

Reply:
Done!